config/config.go

```diff
@@ -203,13 +202,17 @@ func Get(cf string, ec *Extended, logger zap.Logger) (*AppConfig, error) {
 
 	cfg := AppConfig{
 		Logging:                b.Logging,
-		Verbose:                b.Verbose,
 		PIDPath:                b.Arachne.PIDPath,
 		Orchestrator:           b.Arachne.Orchestrator,
 		StandaloneTargetConfig: b.Arachne.StandaloneTargetConfig,
 		Metrics:                mc,
 	}
 
+	if cfg.Orchestrator.Enabled && cfg.StandaloneTargetConfig == "" {
```

Comment:
Should this be if NOT cfg.Orchestrator.Enabled && cfg.STandaloneTargetConfig == "" ?

(It seems StandaloneTargetConfig is only needed if orchestrator mode is disabled.